server/src/main/java/io/spine/server/integration/MessageChannel.java

```diff
@@ -20,20 +20,19 @@
 package io.spine.server.integration;
 
 /**
- * A channel dedicated to exchanging the messages of a single message type.
- *
- * <p>The class of message serves as a channel key.
+ * A channel dedicated to exchanging the messages.
```

Comment:
Please think of something else. This description is duplicating the class title.